src/components/Tokens/TokenDetails/TransactionsTable.tsx

```diff
@@ -0,0 +1,118 @@
+import { Table } from 'components/Table'
+import { LinkCell, TextCell } from 'components/Table/Cells'
+import { useActiveLocale } from 'hooks/useActiveLocale'
+import { useMemo } from 'react'
+import { Column } from 'react-table'
+import { useTheme } from 'styled-components'
+import { shortenAddress } from 'utils/addresses'
+import { NumberType, useFormatter } from 'utils/formatNumbers'
+
+import { mockSwapData } from './mockData'
```

Comment:
😬

Reply:
Add TODOs wherever you're using MockData to be easier to remove later when BE implementation is available.

Reply:
Added